crates/typst-library/src/visualize/image/mod.rs

```diff
@@ -94,6 +94,10 @@ pub struct ImageElem {
     /// ```
     #[default(ImageFit::Cover)]
     pub fit: ImageFit,
+
+    /// TODO: Add docs
+    #[default(false)]
+    pub eval: bool,
```

Comment:
What do you think about renaming it to eval-text?

Reply:
I like the current shorter name better, but in principle I would be fine with changing it.